monai/networks/layers/spatial_transforms.py

```diff
@@ -454,6 +455,12 @@ def __init__(
             reverse_indexing: whether to reverse the spatial indexing of image and coordinates.
                 set to `False` if `theta` follows pytorch's default "D, H, W" convention.
                 set to `True` if `theta` follows `scipy.ndimage` default "i, j, k" convention.
+            zero_centered: whether the affine is applied to coordinates in a zero-centered value range,
+                Default to `None`. With `zero_centered=True`, for example, the center of rotation will be the
```

Comment:
I don't get the behavor of `None`, could you please help describe it in the doc-string?

Thanks.